Newsletter-224: Translate to Hindi

shaavan opened the pull request.

bitschmidty commented on _posts/hi/newsletters/2022-11-02-newsletter.md:
```diff
@@ -0,0 +1,211 @@
+  [कस्बों की स्थिरता] लेनदेन रिले और mempool स्वीकृति के लिए Bitcoin Core की नीतियों को
+  कॉन्फ़िगर करना आसान बनाने के परिणामों के बारे में, जैसे कि `mempoolfullrbf` विकल्प के योग द्वारा
+  किया गया था Bitcoin Core की विकास शाखा (see न्यूज़लेटर्स [#205][news205 rbf],
+  [#208][news208 rbf], [#222][news222 rbf], और [#223][news223 rbf])।
```
Can you make the small change here so we can link to the 223/hi version?

https://github.com/bitcoinops/bitcoinops.github.io/pull/923#discussion_r1018199424

shaavan commented:
Updated from [86ced1d ](https://github.com/bitcoinops/bitcoinops.github.io/commit/86ced1d10005492b99a500350968149c81bd579e)to [f28cc71](https://github.com/bitcoinops/bitcoinops.github.io/commit/f28cc71a487dc439be4a8b674d0104f545106b5d)

Addressed @bitschmidty comments

- Updated link from `en` to `hi`
- Make production is currently failing because of hi-223 links. It should pass after hi-223 gets merged.

bitschmidty reviewed (commented):
I pushed some updates to link anchors as well as duplicate reference links in the footer
